recipe/meta.yaml

```diff
@@ -1,69 +1,176 @@
 {% set name = "jupytercad" %}
-{% set version = "0.3.3" %}
+{% set version = "1.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 2dd55491d06e988b910cb5433d4c4c7080c8a9e8e2778aa4d7e86db6ae19c63f
+  - url: https://pypi.io/packages/source/j/jupytercad_core/jupytercad_core-{{ version }}.tar.gz
+    sha256: d8fa5b4bebfbecae7f9e16e01ff39ae07324516ed1cad6357c357cabfc570d02
+    folder: jupytercad_core
+  - url: https://pypi.io/packages/source/j/jupytercad_lab/jupytercad_lab-{{ version }}.tar.gz
+    sha256: 385882a2ade248dbcd79fe870d90f90cf0d3e3d75761446ff44f528999323612
+    folder: jupytercad_lab
+  - url: https://pypi.io/packages/source/j/jupytercad_app/jupytercad_app-{{ version }}.tar.gz
+    sha256: bfc98439ad4474f6f2c6100df5f9bd03a0af858adfd79538b55dde40152b163b
+    folder: jupytercad_app
+  - url: https://pypi.io/packages/source/j/jupytercad/jupytercad-{{ version }}.tar.gz
+    sha256: 4938583bde5a304569970301a04af11c6221d5364543a38ffde4329307a45e66
+    folder: jupytercad
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
```

Comment:
I might try putting the `noarch: python` into each of the recipes. Just a wild guess though :/